web/src/app/administration/components/workbasket-access-items/workbasket-access-items.component.ts

```diff
@@ -71,13 +84,10 @@ export class WorkbasketAccessItemsComponent implements OnChanges, OnDestroy {
     this.customFields$ = this.accessItemsCustomization$.pipe(getCustomFields(customFieldCount));
   }
 
-  setAccessItemsGroups(accessItems: Array<WorkbasketAccessItems>) {
-    const AccessItemsFormGroups = accessItems.map(accessItem => this.formBuilder.group(accessItem));
-    AccessItemsFormGroups.forEach(accessItemGroup => {
-      accessItemGroup.controls.accessId.setValidators(Validators.required);
+  ngAfterViewInit() {
+    this.inputs.changes.subscribe(next => {
+      if (this.added) next.last.nativeElement.focus();
     });
```

Comment:
will this somehow collide with the function `focusNewInput`?

Reply:
i tested, if this could perhaps replace it, but it does not interact with it, as the input, that is used in focusNewInput is not in the domain of the workbasketAccessItems Component